### PR TITLE
data:thunderbirdceg

### PR DESCRIPTION
**Pull request by thunderbirdceg**

### CODE REVIEW

File: data/thunderbirdceg.json
Comment (Njong392) [inline data/thunderbirdceg.json:9]: Unfortunately, this isn't a valid LinkedIn link. Add the URL to your LinkedIn profile in the `url` field.

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you for contributing to LinkFree :tada: Please check out the review changes suggested.

File: data/thunderbirdceg.json
Comment (Njong392) [inline data/thunderbirdceg.json:10]: Unfortunately, the LinkedIn url you put here doesn't link to any profile. Add the link to your LinkedIn profile
```suggestion
      "url": "<paste in valid LinkedIn link here>",
```

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: data/thunderbirdceg.json
Comment (thunderbirdceg) [reply data/thunderbirdceg.json]: I changed the linkedin url in my repo.
Is it ok now

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Njong392 — APPROVED** [review]
Looks all good now!

**Comment by thunderbirdceg** [comment]
How to merge the PR to main branch..pls guide

**Comment by eddiejaoude** [comment]
@thunderbirdceg you cannot merge, only the maintainers can - I will merge when the checks finish successful (GitHub Actions)

**Comment by thunderbirdceg** [comment]
Hi Eddie..got it..
Thanks this is a good way for me to get started on open source. Going to go full time like you :)

**Comment by eddiejaoude** [comment]
Awesome, yes do it 👍 

You can review other people's pull requests also as a contributor (but only maintainers can merge)

May I suggest improving your GitHub profile, like profile picture, bio etc - we can chat about this in the EddieHub Discord http://discord.eddiehub.org see you there

**Pull request merged**
